okhttp-testing-support/src/main/kotlin/okhttp3/okio/LoggingFilesystem.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.okio
+
+import okio.ExperimentalFileSystem
+import okio.FileSystem
+import okio.ForwardingFileSystem
+import okio.Path
+import okio.Sink
+import okio.Source
+
+@OptIn(ExperimentalFileSystem::class)
+class LoggingFilesystem(fileSystem: FileSystem) : ForwardingFileSystem(fileSystem) {
```

Comment:
You could probably override `onPathParameter()` for all the single-argument methods
```
      override fun onPathParameter(path: Path, functionName: String, parameterName: String): Path {
        println("$functionName($parameterName=$path)")
        return path
      }
```